src/cow-react/modules/swap/containers/Row/RowFee/index.tsx

```diff
@@ -71,14 +81,16 @@ export function RowFee({ trade, fee, feeFiatValue, allowsOffchainSigning, showHe
     const feeAmountWithCurrency = `${smartFeeTokenValue} ${formatSymbol(feeCurrencySymbol)} ${
       isEthFLow ? ' + gas' : ''
     }`
-    const feeToken = smartFeeTokenValue ? feeAmountWithCurrency : '🎉 Free!'
+    console.log(smartFeeTokenValue, smartFeeFiatValue)
```

Comment:
I think the console.log should be removed

Reply:
Great point - removing it